airflow/providers/openlineage/sqlparser.py

```diff
@@ -116,19 +117,26 @@ def from_table_meta(
     return Dataset(namespace=namespace, name=name if not is_uppercase else name.upper())
 
 
-class SQLParser:
+class SQLParser(LoggingMixin):
```

Comment:
Those changes seem pretty unrelated - could we extrac them to a separate PR maybe?

Reply:
Split those into https://github.com/apache/airflow/pull/40200